src/api/trace.zig

```diff
@@ -1,9 +1,91 @@
+const std = @import("std");
+
 pub const Tracer = @import("trace/tracer.zig").Tracer;
 pub const TracerProvider = @import("trace/provider.zig").TracerProvider;
 pub const TracerConfig = @import("trace/config.zig").TracerConfig;
 
+pub const TraceID = struct {
+    value: [16]u8,
+
+    const Self = @This();
+
+    pub fn init(value: [16]u8) Self {
+        return .{
+            .value = value,
+        };
+    }
+
+    pub fn isValid(self: Self) bool {
+        return Self.isValidValue(self.value);
+    }
+
+    pub fn isValidValue(value: [16]u8) bool {
+        for (value) |item| {
+            if (item != 0) {
+                return true;
+            }
```

Comment:
Do I understand correctly that the only invalid value is an array of 16 `u8`s with all elemets set to 0?

Reply:
yes.
the specification about it is defined in https://opentelemetry.io/docs/specs/otel/trace/api/#isvalid.